pkg/placement/where-resolver.go

```diff
@@ -170,12 +174,26 @@ func (wr *whereResolver) process(ctx context.Context, item queueItem) bool {
 	logger := klog.FromContext(ctx)
 	cluster := item.Cluster
 	epName := item.Name
-	objName := item.toExternalName()
-	sps, err := wr.spsLister.Cluster(cluster).Get(epName)
+	sps, err := wr.spsLister.Get(epName)
 	if err != nil && !k8sapierrors.IsNotFound(err) {
 		logger.Error(err, "Failed to fetch SinglePlacementSlice from local cache", "cluster", cluster, "epName", epName)
 		return true // I think these errors are not transient
 	}
+	//change to consumer SpaceID
+	_, spsOriginalName, kbSpaceID, err := kbuser.AnalyzeObjectID(sps)
+	if err != nil {
+		logger.Error(err, "Object does not appear to be a provider's copy of a consumer's object")
+		return true
+	}
+	spaceID := wr.kbSpaceRelation.SpaceIDFromKubeBind(kbSpaceID)
+	if spaceID == "" {
+		logger.Error(nil, "Failed to get consumer space ID from a provider's copy")
```

Comment:
Since this is expected to be a transient condition, `logger.Error` is more alarming than ideal.